From c16c951c019f968ac70d93faeab8c3c5fa8c4b22 Mon Sep 17 00:00:00 2001
From: Bryan Lai <bryanlais@gmail.com>
Date: Wed, 28 Feb 2024 12:40:39 +0800
Subject: [PATCH] python2Packages.wcwidth: fix build

As of version 0.2.13 upstream still supports python2. In the future, this
package should be dropped or pinned to the last working version after the
final release for python2.
---
 pkgs/development/python2-modules/wcwidth/default.nix | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/pkgs/development/python2-modules/wcwidth/default.nix b/pkgs/development/python2-modules/wcwidth/default.nix
index 023b92a86907..87d3f3848da1 100644
--- a/pkgs/development/python2-modules/wcwidth/default.nix
+++ b/pkgs/development/python2-modules/wcwidth/default.nix
@@ -6,5 +6,12 @@ wcwidth.overridePythonAttrs(oldAttrs: {
   propagatedBuildInputs = oldAttrs.propagatedBuildInputs or [] ++ [
     backports-functools-lru-cache
   ];
-})
 
+  /**
+    As of version 0.2.13 upstream still supports python2. In the future, this
+    package should be dropped or pinned to the last working version after the
+    final release for python2. See:
+      https://github.com/jquast/wcwidth/pull/117#issuecomment-1946609638
+  */
+  disabled = false;
+})
-- 
2.43.1

